managed/services/backup/pitr_timerange_service.go

```diff
@@ -181,7 +181,7 @@ func (s *PBMPITRService) ListPITRTimeranges(ctx context.Context, storage Storage
 // trimTimelines adds one second to the Start value of every timeline record. Required to fit PBM values.
 func trimTimelines(timelines []Timeline) {
 	for i := range timelines {
-		timelines[i].Start += 1
+		timelines[i].Start += u1 //nolint:revive
```

Comment:
this change generates error below
```
Error: 24 Jan 24 23:32 UTC ERR Error parsing file error="-: # github.com/percona/pmm/managed/services/backup\n./pitr_timerange_service.go:184:25: undefined: u1" dry-run=false version=v2.39.1
```

Reply:
I have made the requested changes please let me know if any further changes are required